swift.go

```diff
@@ -239,10 +242,10 @@ func (c *Connection) setDefaults() {
 		c.Retries = DefaultRetries
 	}
 	if c.ConnectTimeout == 0 {
-		c.ConnectTimeout = 10 * time.Second
+		c.ConnectTimeout = 60 * time.Second
```

Comment:
sure, as our swift server is far away from my working pc, so need to use a larger timeout value during development , i think we do not need this code merged